<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="dateEdit_maxDay">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_minDay">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_visualizar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualizar por</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>561</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_chart"/>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget_periodo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>296</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_import">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_periodo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Periodo</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_dia">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DÃ­a actual</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_export">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Exportar</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionConfigurar"/>
    <addaction name="actionExportar_datos"/>
   </widget>
   <addaction name="menuOpciones"/>
  </widget>
  <action name="actionConfigurar">
   <property name="text">
    <string>Configurar</string>
   </property>
  </action>
  <action name="actionExportar_datos">
   <property name="text">
    <string>Exportar datos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
